### PR TITLE
Feature/conversations

### PR DESCRIPTION
**Pull request by peter7z**
The home page now shows matches conversations.
screen shot 2018-10-16 at 11 53 32 am

### CODE REVIEW

File: src/components/common/Conversation.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { object } from 'prop-types';
+
+import profilePic from '../../img/guy.svg';
+
+const Conversation = ({
+  conversation: { unreadMessages, user, lastMessage, topicIcon }
+}) => (
+  <div className="row conversation">
+    {unreadMessages != 0 &&
```
Comment (felipegarcia92) [inline src/components/common/Conversation.js:10]: You can do `unreadMessages &&`

File: src/reducers/conversationsReducer.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { fromJS } from 'immutable';
+
+import { GET_CONVERSATION_SUCCESS } from '../actions/actionTypes';
+
+const initialState = fromJS([]);
+
+const topics = (state = initialState, { type, conversations }) => {
```
Comment (felipegarcia92) [inline src/reducers/conversationsReducer.js]: I think you forgot to rename the reducer here after pasting 😂

**Review by felipegarcia92 — CHANGES_REQUESTED** [review]

File: src/components/common/Conversation.js (hunk 1 above)
Comment (peter7z) [reply src/components/common/Conversation.js:10]: It's a string, so when "0" evaluates to true.

File: src/reducers/conversationsReducer.js (hunk 2 above)
Comment (peter7z) [reply src/reducers/conversationsReducer.js]: Right, fixed it.

**Review by felipegarcia92 — APPROVED** [review]

**Review by juanlacu — APPROVED** [review]
Good one!

**Pull request closed without merging**